numpy/random/mtrand/mtrand.pyx

```diff
@@ -4888,6 +4889,247 @@ cdef class RandomState:
 
         return multin
 
+    def _multivariate_hypergeometric1(self, ndarray colors,
```

Comment:
For testing and timing, I had exposed the two private functions at the end of `mtrand.pyx` as:

    multivariate_hypergeometric1 = _rand._multivariate_hypergeometric1
    multivariate_hypergeometric2 = _rand._multivariate_hypergeometric2

But since they aren't public in the version in the pull request, I'll change them to `cdef`.